action/protocol/execution/evm/evmstatedbadapter_test.go

```diff
@@ -812,13 +865,13 @@ func TestSortMap(t *testing.T) {
 		for i := 0; i < size; i++ {
 			stateDB.RevertToSnapshot(sn)
 			s := ""
-			if stateDB.sortCachedContracts {
-				for _, addr := range stateDB.cachedContractAddrs() {
-					c := stateDB.cachedContract[addr]
+			if stateDB.disableSortCachedContracts {
+				for _, c := range stateDB.cachedContract {
```

Comment:
IMO it's not necessary to flip the flag? it increases the risk of making accidental error, and make it hard to interview
if we really want to do it, better do it in a separate PR, since this is not related to zero-nonce